vendor/starkware/README.md

```diff
@@ -0,0 +1,3 @@
+# A collection of tools to support the Stark Exchange
+
+https://github.com/starkware-libs/cairo-lang/tree/master/src/starkware
```

Comment:
Contains more up-to-date code in comparison with https://github.com/starkware-libs/starkex-resources